recipes/openssl/3.x.x/conanfile.py

```diff
@@ -55,6 +56,7 @@ class OpenSSLConan(ConanFile):
         "no_md2": [True, False],
         "no_md4": [True, False],
         "no_mdc2": [True, False],
+        "no_module": [True, False],
```

Comment:
How is this option used in the recipe itself? I would expect something else to change in this `conanfile.py`

Reply:
> How is this option used in the recipe itself?

All options that don't need special treatment are handled in the _for_ loop in lines 438/440 of the method `_configure_args` in openssl's _conanfile.py_

In this case, the option `openssl:no_module=True` would be translated into an argument _no-module_ for the configure call

Reply:
Thanks for the clarification